duke/src/main/java/AddCommand.java

```diff
@@ -0,0 +1,149 @@
+import java.text.SimpleDateFormat;
```

Comment:
It might be good to package your classes into relevant packages such as a Command package, a Components package for your Ui, Parser etc.